client/webserver/site/src/js/markets.ts

```diff
@@ -601,19 +631,16 @@ export default class MarketsPage extends BasePage {
   setOrderVisibility () {
     const page = this.page
     if (this.isLimit()) {
-      Doc.show(page.priceBox, page.tifBox, page.qtyBox, page.maxBox)
+      Doc.show(page.priceBox, page.tifBox, page.qtyBox, page.maxBox, page.rateStepBox, page.orderTotalPreview)
       Doc.hide(page.mktBuyBox)
-      this.previewQuoteAmt(true)
     } else {
-      Doc.hide(page.tifBox, page.maxBox, page.priceBox)
+      Doc.hide(page.priceBox, page.tifBox, page.maxBox, page.rateStepBox, page.orderTotalPreview)
```

Comment:
Initially I wanted to keep the scope of this PR to improving limit-order form, but since that didn't quite work out this way and I changed market-order form already - might as well address it too here.

I've created split event-handlers on market order form (mostly for sell side) and now also validate lot/quantity inputs the same way limit order form does; also added "Total: ~ estimate" the user will receive.